R/qenv-eval_code.R

```diff
@@ -61,7 +63,8 @@ setMethod("eval_code", signature = c(object = "qenv.error"), function(object, co
     x <- withCallingHandlers(
       tryCatch(
         {
-          eval(current_call, envir = object@.xData)
+          .Last.value <- eval(current_call, envir = object@.xData)
+          attr(object@.xData, ".Last.value") <- .Last.value
```

Comment:
Comparing to base - `.Last.value` isn't an attribute but an object of the .GlobalEnv

Reply:
Hmmm, even worse. `.Last.value` is an object in `base` and not assigned to any environment :/